class_linux.go

```diff
@@ -197,9 +217,10 @@ func (h *Handle) ClassList(link Link, parent uint32) ([]Class, error) {
 		}
 
 		base := ClassAttrs{
-			LinkIndex: int(msg.Ifindex),
-			Handle:    msg.Handle,
-			Parent:    msg.Parent,
+			LinkIndex:  int(msg.Ifindex),
+			Handle:     msg.Handle,
+			Parent:     msg.Parent,
+			Statistics: NewClassStatistics(),
```

Comment:
Given `tcstats.Statistics` is a pointer, I do not see a need to initialize an empty stats structure.
Caller will anyway check for non `nil` before accessing a pointer parameter.

later on you can just
```
base.Statistics, err := parseTcStats(attr.Value)
if err != nil {
    return nil, err
}
```

If your idea is to always return a stats structure, maybe empty, then it should not be a pointer.
I prefer current solution as a pointer (nil, in case no stats), up to you.

Reply:
That makes sense. Fixed.